src/main/java/core/basesyntax/fileservise/ReaderServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.fileservise;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
+public class ReaderServiceImpl implements ReaderService {
+
+    @Override
+    public List<String> read(String filePath) {
+        File file = new File(filePath);
+        List<String> dateFromFile;
+
+        try {
+            dateFromFile = Files.readAllLines(file.toPath());
```

Comment:
```suggestion
            return Files.readAllLines(Path.of(filePath));
```
there is no need to create file and then convert it to path

Reply:
done